### PR TITLE
Bulk change: move double quotes outside tokens

### PR DESCRIPTION
**Pull request by glenacota**
Changes in this PR move all the double quotes placed inside a token outside the same token.
The issue, as well as the agreed resolution, is described in #4363: "Token inconsistency: quotes inside/outside tokens".

To ease a bit the review, I made each commit a logically separated unit of pages.

### CODE REVIEW

File: pages.ko/common/banner.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# banner
-
-> 주어진 인자를 큰 ASCII art로 출력.
-
-- 텍스트 메시지를 큰 배너로 출력(따옴표는 선택 사항):
```
Comment (navarroaxel) [inline pages.ko/common/banner.md:5]: Why this text was changed?

File: pages.zh/windows/eventcreate.md [hunk 2]
```diff
@@ -17,4 +17,4 @@
 
 - 在远程计算机的事件日志中创建事件:
 
-`eventcreate /s {{主机名}} /u {{用户名}} /p {{密码}} /t {{类型}} /id {{id}} /d "{{消息"}}`
+`eventcreate /s {{主机名}} /u {{用户名}} /p {{密码}} /t {{类型}} /id {{id}} /d "{{消息}}"`
```
Comment (navarroaxel) [inline pages.zh/windows/eventcreate.md:20]: Nice catch!

File: pages/common/printf.md [hunk 3]
```diff
@@ -4,20 +4,20 @@
...
 
 - Print a text message composed with environment variables:
 
-`printf {{"var1: %s\tvar2: %s\n"}} {{"$VAR1"}} {{"$VAR2"}}`
+`printf "{{var1: %s\tvar2: %s\n}}" "{{$VAR1}}" "{{$VAR2}}"`
 
 - Store a formatted message in a variable (does not work on zsh):
 
-`printf -v {{myvar}} {{"This is %s = %d\n" "a year" 2016}}`
+`printf -v {{myvar}} "{{This is %s = %d\n" "a year" 2016}}`
```
Comment (navarroaxel) [inline pages/common/printf.md:23]: This doesn't look good

File: pages/linux/dunstify.md [hunk 4]
```diff
@@ -5,15 +5,15 @@
...
 
 - Show a notification with specified urgency:
 
-`dunstify {{"Title"}} {{"Message}}" -u {{low|normal|critical}}`
+`dunstify "{{Title}}" "{{Message}}" -u {{low|normal|critical}}`
 
 - Specify a message ID (overwrites any previous messages with the same ID):
 
-`dunstify {{"Title"}} {{"Message"}} -r {{123}}`
+`dunstify "{{Title}}" "{{Message}}" -r {{123}}`
```
Comment (navarroaxel) [inline pages/linux/dunstify.md:16]: Nice catch!

File: pages/windows/eventcreate.md [hunk 5]
```diff
@@ -18,4 +18,4 @@
 
 - Create an event in a remote machine's event log:
 
-`eventcreate /s {{hostname}} /u {{username}} /p {{password}} /t {{type}} /id {{id}} /d "{{message"}}`
+`eventcreate /s {{hostname}} /u {{username}} /p {{password}} /t {{type}} /id {{id}} /d "{{message}}"`
```
Comment (navarroaxel) [inline pages/windows/eventcreate.md:21]: Cool!

**Review by navarroaxel — COMMENTED** [review]
Hey @glenacota Good job!

File: pages.ko/common/banner.md (hunk 1 above)
Comment (glenacota) [reply pages.ko/common/banner.md:5]: wow, I have no idea! those changes are not caught in my IDE, when compared to master

File: pages/common/printf.md (hunk 3 above)
Comment (glenacota) [reply pages/common/printf.md:23]: I knew that would have happened 😅  thanks!

File: pages.zh/windows/eventcreate.md [hunk 6]
```diff
@@ -17,4 +17,4 @@
 
 - 在远程计算机的事件日志中创建事件:
 
-`eventcreate /s {{主机名}} /u {{用户名}} /p {{密码}} /t {{类型}} /id {{id}} /d "{{消息"}}`
```
Comment (schneiderl) [inline pages.zh/windows/eventcreate.md:20]: wow that was looking terrible haha

**Review by schneiderl — APPROVED** [review]
It all looks good apart from the previous comments. 
Thank you for the pull request. Very nice catch.

File: pages.ko/common/banner.md (hunk 1 above)
Comment (navarroaxel) [reply pages.ko/common/banner.md:5]: Looks like you have added an invisible char, but Idk what.
Screenshot_20201003_161908

Comment (waldyrious) [reply pages.ko/common/banner.md:5]: `^M` is the [carriage return character](https://en.wikipedia.org/wiki/Carriage_return) (the CR in CRLF), usually represented as `\r`. You might want to make sure your editor (or [git](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_core_autocrlf)) is not configured to make changes to them. Usually only the line feed character, `\n`, is expected as a line terminator.

**Review by navarroaxel — APPROVED** [review]

**Review by waldyrious — APPROVED** [review]
Looks good, thanks @glenacota!

File: pages.ko/common/banner.md (hunk 1 above)
Comment (bl-ue) [reply pages.ko/common/banner.md]: I know this is old, but I'd like to say (for history's sake), I fixed all of this in #5136.  It looks like @Proscream commited a few CRLF files and those were always revert when anyone tried to modify the files.  I'll resolve this convo now too.

**Pull request merged**